### DISCUSSION TITLE
Suggestion: Add hint to time tracking

### DISCUSSION THREAD

**Discussion by Zeioth** [Manjaro-Sway/manjaro-sway#258] [Ideas]
It might be nice to tell new users what zeit is, with an informative tooltip

![screenshot_2022-02-07_05-30-44_244743951](https://user-images.githubusercontent.com/3357792/152724785-21242285-b708-45c4-af1f-5cd6ff007c57.png)

    # Code from /usr/share/sway/scripts/zeit.sh
    tooltip=$tracking'\r(zeit task tacker)'

**Comment by Mathew-D** [comment]
Can you put it in PR for this as good tool-tips are always important.

**Comment by boredland** [comment]
Has been done!
